### PR TITLE
Bugfix/11 incorrect path

### LINKED ISSUE #11: Incorrect path for conventional-changelog
https://github.com/bumped/bumped-changelog/blob/master/lib/index.coffee#L8

```
Error: spawn C:\Users\zenobius\Projects\Mine\angular-named-routes\node_modules\bumped-changelog\node_modules\.bin\conventional-changelog ENOENT
  at exports._errnoException (util.js:873:11)
  at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
  at onErrorNT (internal/child_process.js:344:16)
  at nextTickCallbackWith2Args (node.js:442:9)
  at process._tickCallback (node.js:356:17)
```

Perhaps it should be : 

```
...
conventionalChangelog = 'node_modules/.bin/conventional-changelog-cli'
...
```

### PR DESCRIPTION
**Pull request by airtonix**
Fixes #11

### CODE REVIEW

File: lib/index.coffee [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-spawn      = require('child_process').spawn
+spawn      = require('cross-spawn')
```
Comment (Kikobeats) [inline lib/index.coffee]: can you remove `(` and `)`  ? 😄

File: package.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "coffee-script": "~1.11.0",
     "conventional-changelog-cli": "~1.2.0",
+    "cross-spawn": "^4.0.2",
```
Comment (Kikobeats) [inline package.json:29]: I prefer `~` instead of `^`

**Comment by Kikobeats** [comment]
so, confirm me, the problem is:
- the dep name is incorrect.
- that are you running this on windows and path works different.

**Comment by Kikobeats** [comment]
One thing, I'm using spawn in more core part of bumped:
- https://github.com/bumped/bumped/blob/27cc25d8feb2fac8673076f3d07896aedca023db/lib/Bumped.plugin.coffee#L7
- https://github.com/bumped/bumped-terminal/blob/c3c1e266bc42f25310791fc4e395a0a3eb27199a/lib/index.coffee#L3

Do you think that also is necessary unify this code?

**Comment by airtonix** [comment]
Yeah seems on windows it's not correctly executing the (existing) `conventional-changelog.cmd`.

**Comment by airtonix** [comment]
@Kikobeats `crossspawn` has `spawn.sync` although I'm unsure as to your use of `execspawn` (seemingly for streams?)

But I just executed a `npm run release -- patch` and no problems that I noticed yet.

I'd leave them for now.

**Comment by Kikobeats** [comment]
yes, the first case could be replace easily. (do you want make a PR? 😋)

The second case, you're right, is oriented in a stream solution, I don't know if we can do something

**Comment by Kikobeats** [comment]
Oh, I remember: I use `execspawn` because `child_process.spawn`doesn't work well when process exit with code status. Need to test again using `crossspawn`.

**Comment by Kikobeats** [comment]
I checked and maybe the problem is how windows resolve the symbolic link:

``` bash
~/Projects/bumped/bumped-changelog/node_modules/.bin master
❯ l
36 Mar 23  2016 conventional-changelog -> ../conventional-changelog-cli/cli.js
```

Can you test this?

``` coffee
conventionalChangelog = 'node_modules/conventional-changelog-cli/cli.js'
```

I prefer avoid a dependency if I can.

**Comment by Kikobeats** [comment]
I'm going to apply my last comment, open the issue again if it doesn't work fine.

**Pull request closed without merging**